.github/workflows/docs.yml

```diff
@@ -30,3 +30,12 @@ jobs:
           python -W ignore -m pip install -r requirements-all.txt
       - name: Build docs
         run: sphinx-build docs/source docs/build/html -W --keep-going -j auto
+      - name: Upload docs
+        uses: actions/upload-artifact@v3
+        with:
+          name: built docs
+          retention-days: 7
+          path: |
```

Comment:
Can you add a short comment on what this does?

Reply:
the days? or the path?

Reply:
The path